src/content/guides/using-persistent-volumes-on-aws-with-efs-csi-driver/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Using persistent volumes with the EFS CSI driver on AWS"
+description: "Tutorial on how to use persistent volumes with the EFS CSI driver on a cluster running on Amazon Web Services with EFS"
+type: page
+weight: 50
+tags: ["tutorial"]
+user_questions:
+  - How do I install the EFS CSI driver?
+  - How do I provision an EFS instance on AWS?
+  - How do I test the EFS CSI storage class?
+  - How do I use EFS CSI Volumes?
+  - What auto-provisioner for AWS do you recommend? EFS? EBS?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-firecracker
+---
+
+# Using Persistent Volumes on AWS with EFS
```

Comment:
```suggestion
# Using persistent volumes on AWS with EFS
```

Recently we decided to use [sentence style capitalization](https://docs.microsoft.com/de-de/style-guide/text-formatting/using-type/use-sentence-style-capitalization) in our docs, including headlines.

Also technical terms like e. g. "persistent volumes" and "availability zones" should be spelled in lower case. Exception: Code snippets/Literals where the exact case is important, e. g.

> ... the `PersistentVolumeClaim` resource ...

We haven't executed this decision in all out docs yet. However it would be nice if you could do it for this new article.

Reply:
This make sense 👍🏻